app/src/main/java/com/example/concordia_campus_guide/Helper/CurrentLocation.java

```diff
@@ -0,0 +1,60 @@
+package com.example.concordia_campus_guide.Helper;
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.location.Location;
+import android.os.Handler;
+
+
+import com.google.android.gms.location.FusedLocationProviderClient;
+import com.google.android.gms.location.LocationServices;
+
+import androidx.fragment.app.Fragment;
+
+public class CurrentLocation extends Fragment {
```

Comment:
why did you make it extend fragment? It's purpose looks more like a helper class...

Reply:
good catch